mlrun/runtimes/nuclio/serving.py

```diff
@@ -408,6 +409,8 @@ def add_model(
             * **archive**:
             1. If model endpoints with the same name exist, preserve them.
             2. Create a new model endpoint with the same name and set it to `latest`.
+        :param outputs: list of labels for the model, Note: please provide the label column, if provided will override
+                        outputs provided in log_model
```

Comment:
maybe worth add more explicit warning like - 
`list of the model outputs (e.g. labels) ,if provided will override the outputs that been configured in the model artifact, please note that those outputs need to be more equal to the model serving function outputs`